### PR TITLE
DOCS-2073 ServiceNow integration

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/user-guides/settings/integrations/servicenow.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# ServiceNow
+
+You can set up Wallarm to create trouble tickets in ServiceNow when the following events are triggered:
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/servicenow.md:3]: let's add the link to the ServiceNow site

File: docs/latest/user-guides/settings/integrations/servicenow.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+* System related:
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/servicenow.md:5]: make it to be include as we use for email

File: docs/latest/user-guides/settings/integrations/servicenow.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    * High risk
+    * Medium risk
+    * Low risk
+* [Scope (exposed assets)](../../scanner.md) changed: updates in hosts, services, and domains
+
+## Setting up integration
+
+In ServiceNow UI:
+
+1. Get name of your ServiceNow instance.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/servicenow.md:19]: how? let's add the link to the servicenow docs

File: docs/latest/user-guides/settings/integrations/servicenow.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+1. Gen username and password to access the instance.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/settings/integrations/servicenow.md:20]: gen - typo

File: docs/latest/user-guides/settings/integrations/servicenow.md (hunk 2 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/settings/integrations/servicenow.md:5]: It is not an inculde as the list of events is unique.

**Pull request merged**